docs/libs.md

```diff
@@ -60,3 +60,6 @@ you'll need to first convert AsciiMath into LaTeX input, then call KaTeX.
 
 - [@rojer/katex-mini](https://github.com/rojer95/katex-mini): A Wechat Mini Program library that uses KaTeX for TeX math rendering.
 
+### Sphinx Extension
```

Comment:
I'd maybe remove Extension here. (I think "LaTeX" should also be removed in "Canvas LaTeX".)

Reply:
I removed " Extension" from "Sphinx Extension" and also " LaTeX" from "Canvas LaTeX".